### ISSUE TITLE
Establish an accessibility audit checklist / set of criteria

### ISSUE THREAD

**Issue by mark-a-wilson** [bcgov/digital.gov.bc.ca#226]
**User Story**
As an **accessibility auditor**, I need a set of criteria that our product needs to meet so that I can audit it and work with our developers to bring it up to snuff and be able to claim that it meets a standard for accessibility. 

**D.O.D.**
- [x] Review the [Accessibility & Inclusion Toolkit](https://www2.gov.bc.ca/gov/content/home/accessible-government/toolkit)
- [x] Talk to Katrina Summers & Ali Emery (GDX) for guidance on what should be the scope of an "MVP" checklist or set of criteria that will constitute our accessibility audit.
- [x] Create checklist that can be used against any page on the site

**Comment by mark-a-wilson** [comment]
After Gillian's note to us, I connected with Katrina Summers and Ali Emery. They directed me to the **[WCAG 2 Checklist](https://webaim.org/standards/wcag/checklist)**. We are to strive to meet **AA** level — which means that we meet the standard for everything in this checklist marked with an A or AA.

I'm going to translate this checklist into **[this spreadsheet](https://docs.google.com/spreadsheets/d/122aEH7rIJAcbV5o3c3grYkkI_eLE0mQ0RT2_lumzM5I/edit#gid=0
)** so that we can run through the full audit and make notes alongside each item for things that need to be addressed to meet the standard.
